### DISCUSSION TITLE
Search - how to combine smart search, persons and location?

### DISCUSSION THREAD

**Discussion by aisbergde** [immich-app/immich#6180] [Q&A] [answered]
Is it currently possible to search a combination of tagged persons, locations or smart search? Or is the search currently rebuild and we should wait?

examples:
- person a at location x
- (person a or person b) at location x
- (person a and person b) at location x
- person a on a boat (boat as example for smart search)
- person a and tree

How to know what is implemented and possible?
https://immich.app/docs/features/search/
is very short

As soon as something above will be implemented, this should be explained a bit.

**Comment by martabal** [comment, marked as answer]
Not yet, but is should come soon thanks to the latest improvements.
